xml/System.Data.Objects/CurrentValueRecord.xml

```diff
@@ -257,10 +257,18 @@
         <Parameter Name="i" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="i">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="i">The ordinal of the field.</param>
+        <summary>This method is not supported in this release.</summary>
+        <returns>The field value as a <see cref="T:System.Data.Common.DbDataReader" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks  
+This method is not supported in this release.
```

Comment:
Perhaps just "This method is not supported."?

Reply:
I've removed the remarks and just changed the summary to say that it throws the NSE in all cases.